earth_enterprise/src/third_party/gdal/README.txt

```diff
@@ -35,3 +35,11 @@ on a SLES 9 box (compatible with SLES 9,10, RHEL 4,5 and Ubuntu 6,8)
 
 To Build, you will need to build on a SLES 9 box using the scripts found in
```

Comment:
Wow, most of the info in this file extremely out of date and doesn't apply to Open GEE at all.

Reply:
If you decide to clean up the file, you can also rename it to README.md

Reply:
I just did clean up the README.txt and renamed it to README.md